bodhi/tests/server/services/test_updates.py

```diff
@@ -171,6 +171,15 @@ def test_duplicate_update(self, *args):
                                  status=400)
         assert 'Update for bodhi-2.0-1.fc17 already exists' in res
 
+    @mock.patch(**mock_valid_requirements)
+    def test_unpushed_update(self, *args):
+
+        unpushed_update = self.create_update(['whoopsie-1.0.0-1.fc17'])
+        unpushed_update.status = UpdateStatus.unpushed
+        self.db.commit()
+        
+        res = self.app.post_json('/updates/', self.get_update('whoopsie-1.0.0-1.fc17'))
```

Comment:
./bodhi/tests/server/services/test_updates.py:181:9: F841 local variable 'res' is assigned to but never used

We're not checking anything here... we could check that the newly created update has a different alias than the older and that `whoopsie-1.0.0-1.fc17` doesn't belong anymore to the old update.

Reply:
We're implicitly checking that the Update creates and doesn't fail due to the same build being included in an Obsolete Update. I can make this check explicit, and this test certainly fails without the logic added in validators.py